go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -458,8 +458,16 @@ func expectDBClientQueries(t *testing.T, queries []string) {
 			continue
 		}
 		var got string
+		heartbeatRe := regexp.MustCompile(`update _vt.vreplication set time_updated=\d+, transaction_timestamp=\d+ where id=\d+`)
```

Comment:
In this case we are the ones writing the heartbeat queries and they are always written as lower case. If there actually was a query written where that wasn't the case, it would be a sure sign that it was not a heartbeat update query. Therefore, I think it being case sensitive is a better catch method.